Pipfile

```diff
@@ -23,7 +23,7 @@ cf-cell-methods = "==0.1.0"
 Sphinx = "==3.2.1"
 m2r2 = "==0.2.5"
 modelmeta = "*"
-ce = "*"
+ce = {editable = true, path = "."}
```

Comment:
Suggest this go in `[dev-packages]`. Logically, it belongs there because it is required for development, and is not a dependency for installing the package itself (i.e., a package is not a dependency of itself).